docs/banzai_nres/development.rst

```diff
@@ -31,3 +31,10 @@ Testing on Real Data
 --------------------
 Generally, the simplest way to reduce data with BANZAI-NRES is to follow the example reduction on the next page.
 For more complex setups that use RabbitMQ and celery, see the helm chart of deployment details.
+
+Notes on settings
```

Comment:
I'm not sure I want to add this to the docs here. We have not documented any of the other settings here. I think I would put the second half of this an inline comment in the settings.py file.

Reply:
I kind of also feel like this setting should be in the traces.py This is e.g. how the SNR pr does it. We haven't been super consistent, but I kind of want settings.py to have the context settings that you would change running on your local machine vs in production. Reduction settings that basically never get changed can go closer to the code that makes use of them.